site/templates/film/item.jet

```diff
@@ -157,9 +157,9 @@
               <h2>{{i18n("meta_detail_bonus_title")}}</h2>
               {{yield list(itemsPerRow=4, itemLayout="landscape") content}}
                 {{range bonus := film.Bonuses}}
-                  <div class="sub-item">
+                  <s72-show-hide-switcher class="sub-item" target-elem=".synopsis">
```

Comment:
I'd prefer if this was structured more like this:
```
<s72-show-hide-switcher target-elem=".synopsis">
  <div class="sub-item">
    .....
  </div>
</s72-show-hide-switcher>
```
I guess it's not really an issue here because the contents are pre-rendered, but putting classes directly onto Relish component elements has caused unexpected style bugs in the past where a component hasn't rendered anything, but still has margins affecting the DOM etc.

Would also be nice if these wrappers could move into the sub_item.jet.  I assume it's okay to have the same switcher on episodes?

Reply:
- Changed structure to above
- Adjusted CSS accordingly
- Moved elems into subItem jet